config/jobs/README.md

```diff
@@ -139,7 +139,7 @@ periodics:
     path_alias: "sigs.k8s.io/cluster-api-provider-aws"
   spec:
     containers:
-    - image: gcr.io/k8s-testimages/kubekins-e2e:v20200428-06f6e3b-1.15
+    - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20210902-e4567b8-master
```

Comment:
If `k8s-staging-test-infra` is preferred to `k8s-testimages`, should we update references everywhere?

Reply:
That's the next PR, I wanted to avoid burying the little tag cleanups in large search-replace PR

Reply:
Also, this is solely for kubekins right now, not all of the other images have been migrated